core/protocol/ideWebview.ts

```diff
@@ -96,4 +96,5 @@ export type ToWebviewFromIdeProtocol = ToWebviewFromIdeOrCoreProtocol & {
   exitEditMode: [undefined, void];
   focusEdit: [undefined, void];
   focusEditWithoutClear: [undefined, void];
+  showGraniteOnboardingCard: [boolean, void];
```

Comment:
`setShowGraniteOnboardingCard` - all these start with a verb, but the verb here isn't "show" since you can `showGraniteOnboadingCard(false)`

Reply:
done